chat/tests.py

```diff
@@ -24,6 +24,9 @@ def delete_text_channel(self, channel_id):
     def delete_audio_channel(self, channel_id):
         self.audio_channels.remove(channel_id)
 
+    def create_invite_link(self, channel_id):
```

Comment:
We should probably say "url" instead of "link" here, to be extra explicit about the return value.

Reply:
In general, would `create_channel_url` make sense here?